app/views/projects/_listing.html.erb

```diff
@@ -1,33 +1,45 @@
-        </li>
+
+  <div class="row"> 
+    <% if project.image_url %>
+      <div class="col-xs-3"><%= image_tag project.image_url, class: "img-responsive" %></div>
       <% end %>
-      <% documents_count = project.documents.count %>
-      <% if documents_count > 0 %>
-        <li data-toggle="tooltip" data-placement="top"
-            title="<%= documents_count %> related <%= 'document'.pluralize(documents_count) %>">
-          <i class="fa fa-file-text-o"></i> <%= documents_count %>
-        </li>
-      <% end %>
-      <% commit_count = project.commit_count %>
-      <% if commit_count > 0 %>
-        <li data-toggle="tooltip" data-placement="top"
-            title="<%= commit_count %> <%= 'commit'.pluralize(commit_count) %>">
-          <i class="fa fa-github-alt"></i> <%= commit_count %>
-        </li>
-      <% end %>
-    </ul>
-  </div>
-  <div style="display: inline-block; font-size: 1.0em; font-style: italic;">
-    <div>Status: <strong style="font-style: normal;"><%= project.status.upcase %></strong></div>
-    <div><%= "Created #{time_ago_in_words(project.created_at)} ago" %> by <%= link_to project.user.display_name, user_path(project.user) %></div>
-  </div>
-  <p><%= project.description.truncate(250, separator: ' ') %></p>
+  <div class="col-xs-9">
+        
+    <h3><%= link_to project.title, project_path(project) %></h3>
+    <div class="stats">
+      <ul>
+        <% follower_count = project.followers.count %>
+        <% if follower_count > 0 %>
+          <li data-toggle="tooltip" data-placement="top"
+              title="<%= follower_count %> <%= 'member'.pluralize(follower_count) %>">
+            <i class="fa fa-users"></i> <%= follower_count %>
+          </li>
+        <% end %>
+        <% documents_count = project.documents.count %>
+        <% if documents_count > 0 %>
+          <li data-toggle="tooltip" data-placement="top"
+              title="<%= documents_count %> related <%= 'document'.pluralize(documents_count) %>">
+            <i class="fa fa-file-text-o"></i> <%= documents_count %>
+          </li>
+        <% end %>
+        <% commit_count = project.commit_count %>
+        <% if commit_count > 0 %>
+          <li data-toggle="tooltip" data-placement="top"
+              title="<%= commit_count %> <%= 'commit'.pluralize(commit_count) %>">
+            <i class="fa fa-github-alt"></i> <%= commit_count %>
+          </li>
+        <% end %>
+      </ul>
+    </div>
+    <div style="display: inline-block; font-size: 1.0em; font-style: italic;">
+      <div>Status: <strong style="font-style: normal;"><%= project.status.upcase %></strong></div>
+      <div><%= "Created #{time_ago_in_words(project.created_at)} ago" %> by <%= link_to project.user.display_name, user_path(project.user) %></div>
+    </div>
+</div>
+<div class="container-fluid">
+  <div class="row"> <div class="col-xs-0"></div><div class="col-xs-12">
+    <p><%= project.description.truncate(250, separator: ' ') %></p>
+  </div></div>
```

Comment:
I've corrected the formatting, and also double checked every div was closed in show, index, and _form.